### ISSUE TITLE
Provide way to add links outside of item definitions

### ISSUE THREAD

**Issue by bavovanachte** [melexis/sphinx-traceability-extension#57]
Our specific use case:

Want to link SWRQT items (split over multiple files) to ITEST items (all in a single autogenerated file). 

- As the ITEST items are autogenerated, we can't add links in the item definition without adapting the generated file.
- Would like to avoid adding the link in the SWRQT item definition, as these requirements need to be easily reusable for other projects (with a different set of ITEST items)

A proposal would be to add an additional directive, like so:

```
.. item-link::
    :source: SWRQT-flits
    :targets: ITEST-flats
    :type: verified_by
```

This could then be maintained in a different file, easily swappable per project

**Comment by bavovanachte** [comment]
Thoughs @Ben2022 @SteinHeselmans @Letme ?

**Comment by SteinHeselmans** [comment]
notes:

- source and target could be a list of items
- directive doesn't generate any html in output, only adds a relation to the 'database'

**Comment by SteinHeselmans** [comment]
closed by #58
